src/devices/nodon.ts

```diff
@@ -1,11 +1,59 @@
-import {Definition} from '../lib/types';
+import {Definition, Fz, Tz, KeyValueAny} from '../lib/types';
 import * as exposes from '../lib/exposes';
 import * as reporting from '../lib/reporting';
 import extend from '../lib/extend';
 import * as ota from '../lib/ota';
+import * as utils from '../lib/utils';
 const e = exposes.presets;
+const ea = exposes.access;
 import tz from '../converters/toZigbee';
 import fz from '../converters/fromZigbee';
+const manufacturerOptions = {manufacturerCode: 0x128B};
+
+const fzLocal = {
+    fil_pilote_mode: {
```

Comment:
Since this code is duplicated, could you move it to `fromZigbee.ts`/`toZigbee.ts`?